### ISSUE TITLE
Call to animateMarkerToCoordinate on MarkerAnimated does not change the marker position

### ISSUE THREAD

**Issue by ArchieGregory** [react-native-maps/react-native-maps#4013]
# Bug report

## Summary
Calling animateMarkerToCoordinate does not have any effect on the marker, and no error is thrown.
The marker does not move at all.

## Environment info

      react-native: 0.64.3
      react: 17.0.1
      react-native-maps: ^0.29.4 
      expo: 43.0.2
      "typescript": "~4.3.5"
	  Target Platform: android 12

## Steps to reproduce

1. Pass ````animateMarkerToCoordinate````  an object with a latitude and longitude as so ````markerReference.current.animateMarkerToCoordinate(newCoordinate, duration);````
2. Watch the MapView as the marker does not move

### Describe what you expected to happen:

Expected the marker to move from it's location to the new one

### Describe what you actually happens:

The marker does not move

## Reproducible sample code
Small  reproducible version
[Issue with animateMarkerToCoordinate - Snack (expo.dev)](https://snack.expo.dev/@archiegregory/issue-with-animatemarkertocoordinate)

````
export default function App() {
  let markerReference = useRef<MarkerAnimated>();
  const [coordinate, setCoordinate] = useState<AnimatedRegion>(new AnimatedRegion({
    latitude: 51,
    longitude: 0,
    latitudeDelta: 0.0922,
    longitudeDelta: 0.0421,
  }));

  function wait(ms) {
    var start = new Date().getTime();
    var end = start;
    while (end < start + ms) {
      end = new Date().getTime();
    }
  }
  console.log("open")
  function componentWillReceiveProps(nextProps: { latitude?: number; longitude?: number;}) {
    const duration = 500
    
      if (Platform.OS === 'android') {
        if (markerReference.current) {
            markerReference.current.animateMarkerToCoordinate(
            nextProps,
            duration
          );
        }
      } else {
        
        console.log("Not ios")
      }
  }
  let coords = {
    latitude: (48),
    longitude: (2)
  };
  wait(5000);
  console.log("meant to be moving")
  componentWillReceiveProps(coords);
  console.log("should have moved")

  return (
    <View style={styles.container}>
      <MapView 
      style={styles.map} 
      showsUserLocation={true}
      >
        <MarkerAnimated
          ref={marker => { markerReference = marker }}
          coordinate={coordinate}
        />
      </MapView>
    </View>
  );
}
````

**Comment by jasonmichaels** [comment]
This has been fixed w/ PR #4011 @christopherdro

**Comment by ArchieGregory** [comment]
Thank you, am I right in that running this 
````
yarn add git+https://github.com/react-native-maps/react-native-maps.git\#master
````
will allow me to use the latest version of this code, including your PR?
